### DISCUSSION TITLE
What's the recommended workflow for patches when using workspaces ?

### DISCUSSION THREAD

**Discussion by cassepipe** [pnpm/pnpm#6809] [Q&A] [answered]
From what I understood this is the way to go :

1. Go into the relevant project
2. Patch your dependancy
3. Move the generated `"pnpm"` field in `yourproject/package.json` to your root's `package.json`
4. Move the content of `yourproject/patches` to `root/patches`

Is that correct ?

**Comment by await-ovo** [comment, marked as answer]
We can  run`pnpm patch` directly in the workspace root directory, no need to do anything else~.

**Reply by cassepipe** [reply]
Thanks !
